application/dash_application/utility/plot_builder.py

```diff
@@ -361,7 +361,7 @@ def generate_bar(df, criteria, x_fn, y_fn, title_text, yaxis_text):
 
 
 # TODO: Make this more general. Currently it is written for the SARS-CoV-2 view
-def generate_line(df, criteria, x_fn, y_fn, title_text, yaxis_text):
+def generate_line(df, criteria, x_fn, y_fn, title_text, yaxis_text, xaxis_text=""):
```

Comment:
I would make this `xaxis_text=None` and having `'visible': xaxis_text is not None`. This way one could still show the x-axis values and not have a x-axis title.

Regarding the TODO. You can remove it. I mean it did not have all the features of the scatter graph (shape, cutoff lines, etc), but there is no need to make unnecessary work.

Reply:
Both fixed